examples/keras_imagenet_resnet50.py

```diff
@@ -147,10 +147,11 @@
     hvd.callbacks.LearningRateWarmupCallback(warmup_epochs=args.warmup_epochs, verbose=verbose),
```

Comment:
Pass initial_lr to `LearningRateWarmupCallback` as well?

Reply:
Good catch!  Fixed.